src/components/navigation/UserAccounts.js

```diff
@@ -13,49 +19,159 @@ const Wrapper = styled.div`
+    }
+
+    .balance {
+      font-weight: 400;
+    }
+`
+
+const SyncButton = styled.span`
+    background-color: #f8f8f8;
+    border-radius: 50px;
+    border: 2px solid #f8f8f8;
+    color: #0072ce;
+    font-size: 12px;
+    font-weight: 500;
+    padding: 4px 8px;
+
+    :hover, :active {
+    background-color: #F0F0F1;
+    border-color: #F0F0F1;
+    }
+
+    &.dots {
+        color: #0072ce;
+        margin: 0 12px 0 0;
+        padding: 0 12px 0 6px;
+
+        :after {
+            content: '.';
+            animation: link 1s steps(5, end) infinite;
+
+            @keyframes link {
+                0%, 20% {
+                    color: rgba(0,0,0,0);
+                    text-shadow:
+                        .3em 0 0 rgba(0,0,0,0),
+                        .6em 0 0 rgba(0,0,0,0);
+                }
+                40% {
+                    color: #0072ce;
+                    text-shadow:
+                        .3em 0 0 rgba(0,0,0,0),
+                        .6em 0 0 rgba(0,0,0,0);
+                }
+                60% {
+                    text-shadow:
+                        .3em 0 0 #0072ce,
+                        .6em 0 0 rgba(0,0,0,0);
+                }
+                80%, 100% {
+                    text-shadow:
+                        .3em 0 0 #0072ce,
+                        .6em 0 0 #0072ce;
+                }
+            }
         }
     }
 `
 
-const UserAccounts = ({ accounts, accountId, selectAccount }) => (
+const UserAccounts = ({ accounts, accountId, selectAccount, accountsBalance, balance, refreshBalance, getBalance }) => (
     <Wrapper>
-        <Account>
-            {accountId}
-        </Account>
+        <UserAccount
+            accountId={accountId}
+            balance={actionsPending('GET_BALANCE') ? '' : balance?.available}
+            balanceLoading={actionsPending('GET_BALANCE')}
+            refreshBalance={() => (getBalance(), refreshBalance(accountId))}
+            active={true}
+        />
         {accounts.filter(a => a !== accountId).map((account, i) => (
-            <Account key={`link-${i}`} onClick={() => selectAccount(account)} className='additional-account'>
-                {account}
-            </Account>
+            <UserAccount
+                key={i}
+                accountId={account}
+                balance={accountsBalance && accountsBalance[account]?.available}
+                balanceLoading={accountsBalance && accountsBalance[account]?.loading}
+                refreshBalance={() => refreshBalance(account)}
+                active={false}
+                onClick={() => selectAccount(account)}
+            />
         ))}
     </Wrapper>
 )
 
-export default UserAccounts;
\ No newline at end of file
+const UserAccount = ({ accountId, balance, balanceLoading, refreshBalance, active, onClick }) => (
+    <Account className={active ? 'active-account' : 'additional-account'}>
+        <div className='account-data' onClick={onClick}>
```

Comment:
The tap target here only spans the width of the text. This results in nothing happening if I click the space in-between the account name and the "Sync" button. Ideally, the tap target should span the width of the container so the user can click anywhere other than the "Sync" button to switch to a different account.

Reply:
good catch @corwinharrell , it should be good now.